Update fittemplate.md

contsili opened the pull request:
Fixed typo: from cfg.method = 'singlesphere' to cfg.method = 'singleshell'.

robertoostenveld commented:
I have not read that page recently, but eave you considered that (as I remember) the example is actually about fitting a sphere (center and radius) to the polhemus head surface so that the sphere can be used to reshape a template head model (and anatomical MRI) in case no individual anatomy is present?

contsili commented:
Based on discussions with @robertoostenveld & @schoffelen: I fixed typos in the single-shell headmodel subsection and added a single-sphere headmodel subsection.
